src/CMakeLists.txt~

```diff
@@ -0,0 +1,128 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
Also I noticed this is a bad copy of the CMakeLists, isn't it? it looks like the kind of copies I get when not closing properly my text editor